src/main/java/io/aiven/kafka/auth/AivenAclAuthorizerV2.java

```diff
@@ -98,6 +118,13 @@ public void configure(final java.util.Map<String, ?> configs) {
                 watchKeyReference.set(subscribeToAclChanges(configFile));
             }
         }, 0, config.configRefreshInterval(), TimeUnit.MILLISECONDS);
```

Comment:
I think in the future it makes sense to use one executor with 2 threads both for Auditor API and Watch service. Not sure does it makes sense to add such things in this PR.

Reply:
IMO it doesn't make sense. An executor by itself is pretty lightweight, we can create them liberally.